spec.html

```diff
@@ -42955,6 +42966,383 @@ <h1>Properties of _TypedArray_ Instances</h1>
+          1. Let _taRecord_ be MakeTypedArrayWithBufferWitnessRecord(_ta_, ~seq-cst~).
+          1. If IsTypedArrayOutOfBounds(_taRecord_) is *true*, throw a *TypeError* exception.
+          1. Let _len_ be TypedArrayLength(_taRecord_).
+          1. Let _byteOffset_ be _ta_.[[ByteOffset]].
+          1. Let _bytes_ be a new empty List.
+          1. Let _index_ be 0.
+          1. Repeat, while _index_ < _len_,
+            1. Let _byteIndex_ be _byteOffset_ + _index_.
+            1. Let _byte_ be ℝ(GetValueFromBuffer(_buffer_, _byteIndex_, ~uint8~, *true*, ~unordered~)).
+            1. Append _byte_ to _bytes_.
+            1. Set _index_ to _index_ + 1.
+          1. Return _bytes_.
+        </emu-alg>
+      </emu-clause>
+
+      <emu-clause id="sec-writeuint8arraybytes" type="abstract operation">
+        <h1>
+          SetUint8ArrayBytes (
+            _into_: a Uint8Array,
+            _bytes_: a List of byte values,
+          ): ~unused~
+        </h1>
+        <dl class="header"></dl>
+        <emu-alg>
+          1. Let _offset_ be _into_.[[ByteOffset]].
+          1. Let _len_ be the length of _bytes_.
+          1. Let _index_ be 0.
+          1. Repeat, while _index_ < _len_,
+            1. Let _byte_ be _bytes_[_index_].
+            1. Let _byteIndexInBuffer_ be _index_ + _offset_.
+            1. Perform SetValueInBuffer(_into_.[[ViewedArrayBuffer]], _byteIndexInBuffer_, ~uint8~, 𝔽(_byte_), *true*, ~unordered~).
+            1. Set _index_ to _index_ + 1.
+        </emu-alg>
+      </emu-clause>
+
+      <emu-clause id="sec-skipasciiwhitespace" type="abstract operation">
+        <h1>
+          SkipAsciiWhitespace (
+            _string_: a string,
+            _index_: a non-negative integer,
+          ): a non-negative integer
+        </h1>
+        <dl class="header">
+        </dl>
+        <emu-alg>
+          1. Let _length_ be the length of _string_.
+          1. Repeat, while _index_ < _length_,
+            1. Let _char_ be the code unit at index _index_ of _string_.
+            1. If _char_ is neither 0x0009 (TAB), 0x000A (LF), 0x000C (FF), 0x000D (CR), nor 0x0020 (SPACE), then
+              1. Return _index_.
+            1. Set _index_ to _index_ + 1.
+          1. Return _index_.
+        </emu-alg>
+      </emu-clause>
+
+      <emu-clause id="sec-decodebase64chunk" type="abstract operation">
+        <h1>
+          DecodeBase64Chunk (
+            _chunk_: a string,
+            optional _throwOnExtraBits_: a boolean,
+          ): either a normal completion containing a List of byte values, or a throw completion
+        </h1>
+        <dl class="header">
+        </dl>
+        <p>The <dfn id="standard-base64-alphabet">standard base64 alphabet</dfn> is the String whose elements are the code units corresponding to every letter and number in the Unicode Basic Latin block along with *"+"* and *"/"*; that is, it is *"ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/"*.</p>
```

Comment:
```suggestion
        <p>The <dfn id="standard-base64-alphabet">standard base64 alphabet</dfn> is *"ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/"*, the String whose elements are the code units corresponding to every letter and number in the Unicode Basic Latin block along with *"+"* and *"/"*.</p>
```

I think this reads better. Also consider linking "Basic Latin block" to an authoritative source.

Reply:
I think that's actually the best link, but it's no big deal if we don't link it.